geps/overview.md

```diff
@@ -61,18 +61,45 @@ against them. At this stage, the GEP should be targeting the "Implementable"
 stage.
 
 ### 4. Implement the GEP as "Experimental"
+
 With the GEP marked as "Implementable", it is time to actually make those
 proposed changes in our API. In some cases, these changes will be documentation
 only, but in most cases, some API changes will also be required. It is important
 that every new feature of the API is marked as "Experimental" when it is
 introduced. Within the API, we use `<gateway:experimental>` tags to denote
-experimental fields.
+experimental fields. In order to transition a GEP into `Experimental` **the
+graduation criteria for the GEP MUST be filled out**, indicating all of the
+currently known issues and dilemmas that need to be resolved in order to
+graduate to `Standard`.
 
 Before these changes are released, they will also need to be documented.
 GEPs that have not been both implemented and documented before a release
 cut off will be excluded from the release.
 
-### 5. Graduate the GEP to "Standard"
+### 5. Probationary Period
+
+Any GEP in the `Experimental` phase is automatically under a "probationary
+period" where the **maintainers will assess the ability and likelihood that
+the graduation criteria can be resolved within a given time period**. By
+default that time period is **6 months**. Maintainers may select an alternative
+time duration for a probationary period if appropriate, and will document their
```

Comment:
Maybe we should just add the following requirements for GEPs graduating to experimental:

1. Clearly document graduation criteria
2. Propose date after which the GEP will automatically be rejected if graduation criteria is not met

That leaves more of the power in the hands of the person pushing the GEP forward and more closely links the graduation criteria to a timeline.

Reply:
Changed in 2cc4acc93314fe0590d3afe82191ce571f023c6d